napari/utils/theme.py

```diff
@@ -213,13 +213,14 @@ def get_theme(name, as_dict=None):
     if as_dict is None:
         warnings.warn(
             trans._(
-                "Themes were changed to use evented model with Pydantic's color type rather than the `rgb(x, y, z)`. The `as_dict=True` option will be changed to `as_dict=False` in 0.4.15",
+                "The `as_dict` kwarg default to False` since Napari 0.4.17, "
+                "and will become a mandatory parameter in the future.",
```

Comment:
```suggestion
                "The `as_dict` kwarg default to False` since Napari 0.4.17, and will become a mandatory parameter in the future."
```

Reply:
There is no need to have single line strings for `trans._`, but if you really prefere sure.